### DISCUSSION TITLE
Auto select first function argument

### DISCUSSION THREAD

**Discussion by corbkris** [neoclide/coc.nvim#4749] [Q&A] [answered]
Is there a way to auto select the first function argument upon auto completing a function? For example when I autocomplete fmt.Printf(w,a) the cursor is always at the end of the ")" character but I need it to auto select the "w" character.

**Comment by chemzqm** [comment, marked as answer]
It should be auto selected if the inserted text is a well-formed snippet.

**Reply by corbkris** [reply]
I don't think it's a snippet. It's just a function from go's fmt package that gets autocompleted. Creating snippets for every single function would be very time consuming. Is there a way to auto select the first function argument without it being a snippet?

**Reply by chemzqm** [reply]
> would be very time consuming

Should be not

> Is there a way to auto select the first function argument without it being a snippet?

Not yet.
